hypershift-operator/main.go

```diff
@@ -114,7 +120,10 @@ func NewStartCommand() *cobra.Command {
 	cmd.Flags().StringVar(&opts.OpenTelemetryEndpoint, "otlp-endpoint", opts.OpenTelemetryEndpoint, "An OpenTelemetry collector endpoint (e.g. localhost:4317). If specified, OTLP traces will be exported to this endpoint.")
 	cmd.Flags().BoolVar(&opts.EnableOCPClusterMonitoring, "enable-ocp-cluster-monitoring", opts.EnableOCPClusterMonitoring, "Development-only option that will make your OCP cluster unsupported: If the cluster Prometheus should be configured to scrape metrics")
 	cmd.Flags().BoolVar(&opts.EnableCIDebugOutput, "enable-ci-debug-output", false, "If extra CI debug output should be enabled")
-	cmd.Flags().StringVar(&opts.Region, "aws-region", opts.Region, "AWS region in which the operator will create resources")
+	cmd.Flags().StringVar(&opts.AWSRegion, "aws-region", "", "AWS region in which the operator will create resources and in which the --aws-oidc-bucket-name is. Required for AWS guest clusters.")
+	cmd.Flags().StringVar(&opts.AWSOIDCBucketName, "aws-oidc-bucket-name", "", "Name of the bucket in which to store the clusters OIDC discovery information. Required for AWS guest clusters")
```

Comment:
These should align with the install flags right?

Reply:
updated